NHibernate.Caches.Common.Tests/NHibernate.Caches.Common.Tests.csproj

```diff
@@ -3,12 +3,33 @@
   <PropertyGroup>
     <Product>NHibernate.Caches.Common.Tests</Product>
     <Description>Unit tests base for cache providers.</Description>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
     <IsTestProject>true</IsTestProject>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
+    <DefineConstants>NETFX;$(DefineConstants)</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug-NetFx|AnyCPU'">
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release-NetFx|AnyCPU'">
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug-Core|AnyCPU'">
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
Instead of cluttering build configurations, I may have better rewrite the Nant build like it was done in NHibernate.